engine/src/main/scala/cromwell/engine/workflow/workflowstore/InMemoryWorkflowStore.scala

```diff
@@ -76,6 +76,8 @@ class InMemoryWorkflowStore extends WorkflowStore {
     Future.successful(workflowIds.size)
 
   override def switchOnHoldToSubmitted(id: WorkflowId)(implicit ec: ExecutionContext): Future[Unit] = Future.successful(())
+
+  override def findWorkflowsWithAbortRequested(cromwellId: String)(implicit ec: ExecutionContext): Future[Iterable[WorkflowId]] = Future.successful(List.empty)
```

Comment:
I assume this is ok because this `InMemory` version is only used in tests... but in that case could this class be put in `src/test` rather than `src/main`?